ios/Capacitor/Capacitor/Plugins/CapacitorUrlRequest.swift

```diff
@@ -108,14 +108,23 @@ public class CapacitorUrlRequest: NSObject, URLSessionTaskDelegate {
         }
     }
 
+    @available(*, deprecated)
     public func setRequestHeaders(_ headers: [String: String]) {
         headers.keys.forEach { (key: String) in
             let value = headers[key]
-            request.addValue(value!, forHTTPHeaderField: key)
+            request.setValue(value!, forHTTPHeaderField: key)
```

Comment:
```suggestion
            request.addValue(value!, forHTTPHeaderField: key)
```
Since we are keeping the method just to not break things, we should not change the behavior neither